website/docs/d/datasource_compute_address.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
you also need to update website/google.erb if you want your documentation page to show up in the side menu